parameters.yaml

```diff
@@ -15,3 +15,14 @@ parameters:
     description: |
         An ID for the organization to be created is the only required parameter.
         This ID must be unique and match this regular expression: ^[a-z0-9_]{4,30}$
+
+  XIdempotencyKeyHeader:
+    name: X-Idempotency-Key
+    in: header
+    type: string
+    required: false
+    description: |
+      Idempotency key. If given, the request is guaranteed to be executed only
+      once, even if the same request data including the same idempotency key is
```

Comment:
The request can be executed multiple times. This is the trick about this feature. It is more that we ensure to work towards a desired state, which at the end will be the same, no matter how often we try to reach the desired state. At some point we reach it and no further changes will be applied to reasources being created.